### PR TITLE
[HOPSWORKS-3321] Add tutorials endpoint

### PR DESCRIPTION
**Pull request by moritzmeister**

### CODE REVIEW

File: files/default/sql/ddl/3.0.0__initial_tables.sql [hunk 1]
```diff
@@ -2309,4 +2309,4 @@ CREATE TABLE IF NOT EXISTS `validation_result` (
     PRIMARY KEY (`id`),
     KEY (`expectation_id`),
     CONSTRAINT `report_fk_validation_result` FOREIGN KEY (`validation_report_id`) REFERENCES `validation_report` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
-) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
\ No newline at end of file
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
```
Comment (SirOibaf) [inline files/default/sql/ddl/3.0.0__initial_tables.sql:2312]: Revert this change.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: files/default/sql/ddl/3.0.0__initial_tables.sql (hunk 1 above)
Comment (moritzmeister) [reply files/default/sql/ddl/3.0.0__initial_tables.sql]: done

**Review by SirOibaf — APPROVED** [review]
